docs/about/release-notes.md

```diff
@@ -92,6 +92,25 @@ to support such customization.
 
 [blocks]: ../user-guide/styling-your-docs/#overriding-template-blocks
 
+#### Support for dirty builds. (#990)
+
+For large sites the build time required to create the pages can become problematic,
+thus a "dirty" build mode was created. This mode simply compares the modified time
+of the generated html and source markdown. If the markdown has changed since the
+html then the page is re-constructed. Otherwise, the page remains as is. This mode
+may be invoked in both the `mkdocs serve` and `mkdocs build` commands:
+
+```text
+mkdocs serve --dirtyreload
+```
+
+```text
+mkdocs build --dirty
+```
+
+It is important to note that this method for building the pages is for development
+of content only, since the navigation and other links do not get updated.
```

Comment:
Can we add "on other pages" to the end of this sentence (or something similar)? I just want to make it clear that just because the nav is correct on the "updated" page, that does not mean it is on others.